src/context.ts

```diff
@@ -124,7 +124,7 @@ export async function getContext(): Promise<Context> {
   };
 
   core.saveState('isPost', post);
-  core.saveState('composeFile', context.composeFile);
+  core.saveState('composeFiles', context.composeFiles);
```

Comment:
Should we put an explicit `JSON.stringify` in here to match the `JSON.parse` below?

Reply:
This is the part I didn't fully understand. `composeArguments` below doesn't have JSON.stringify but has JSON.parse on reading step. I copied that. But if these state variables are store as strings, yes then stringify() is needed.

Reply:
@sjagoe found it from the core toolkit docs

```
 * @param     value    value to store. Non-string values will be converted to a string via JSON.stringify
```

So `JSON.stringify()` is not needed. This should be correct.

Reply:
`getState` doesn't have automatic conversion so there `JSON.parse()` is needed.